src/live.asp.net/gulpfile.js

```diff
@@ -56,4 +87,8 @@ gulp.task("csslint", function() {
     return gulp.src(paths.css)
         .pipe(csslint())
         .pipe(csslint.reporter());
-});
\ No newline at end of file
+});
+
+gulp.task("build", ["clean", "min", "lib", "csslint", "jshint"]);
```

Comment:
yes, made changes to new commit for all tasks to return streams and dependencies. missed that the clean tasks were not returning this. Nullifies the point of gulp really.